### PR TITLE
mag footer block

### LINKED ISSUE #19: Magazine article footer: Summary of issue
Same view as in the Navigation, see https://github.com/hlxsites/24life/issues/10. 

https://www.24life.com/patricia-moreno-awakens-the-brain-body-connection-to-transform-you/

Screenshot 2023-07-31 at 11 20 55

### PR DESCRIPTION
**Pull request by rrusher**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #19

Test URLs:
- Before: https://main--24life--hlxsites.hlx.live/magazine/volume-5-issue-12
- After: https://mag-footer--24life--hlxsites.hlx.live/magazine/volume-5-issue-12

### CODE REVIEW

**Comment by rrusher** [comment]
Hero still causing issues with page performance.

**Review by synox — APPROVED** [review]
Looks great. Maybe make the magazine cover image a link.

File: blocks/magazine-footer/magazine-footer.css [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+.magazine-footer img {
+  transform: rotate(-6deg);
+  box-shadow: -5px 5px 10px rgba(0 0 0 / 30%);
+  height: auto;
+}
+
+@media (min-width: 900px) {
+  .magazine-footer {
+    display: flex;
+    grid-template-columns: repeat(3, 1fr);
```
Comment (iamitshri) [inline blocks/magazine-footer/magazine-footer.css:53]: grid template columns will not work here
 image

**Review by iamitshri — APPROVED** [review]
Love it!

**Pull request merged**